examples/hello_world/src/main.rs

```diff
@@ -17,14 +17,18 @@ fn main() -> Result<(), eframe::Error> {
 
 struct MyApp {
     name: String,
+    name2: String,
     age: u32,
+    age2: u32,
```

Comment:
please revert the changes in this file. Use `egui_demo_app` (`cargo r`) for a better test of your code!